lib/linguist/languages.yml

```diff
@@ -6501,6 +6499,22 @@ Smithy:
   extensions:
   - ".smithy"
   language_id: 1027892786
+Snakemake:
+  type: programming
+  group: Python
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#419179"
+  extensions:
+  - ".smk"
+  - ".snakefile"
+  filenames:
+  - Snakefile
```

Comment:
I've just remembered that we're going to need a sample file of this name at `samples/Snakemake/filenames/Snakefile` too. I thought we had a test that would catch this, but seems not.

Reply:
done